build.xml

```diff
@@ -229,6 +229,7 @@
         <pathelement location="${libdir}/jbidibc-messages-2.0.18.jar"/>
         <pathelement location="${libdir}/jbidibc-net-serial-over-tcp-2.0.18.jar"/>
         <pathelement location="${libdir}/jbidibc-purejavacomm-2.0.18.jar"/>
+        <pathelement location="${libdir}/jbidibc-jserialcomm-2.0.18.jar"/>
```

Comment:
If purejsvacom is not used anymore, it would be good to remove the reference to the library jbidibc-purejavacomm-2.0.18.jar